fluent/src/intl_memoizer.js

```diff
@@ -0,0 +1,17 @@
+export default class IntlMemoizer {
+  constructor() {
+    this._intls = new WeakMap();
+  }
+
+  get(ctor, locales, opts) {
+    const cache = this._intls.get(ctor) || new Map();
+    const id = {locales, ...opts};
+
+    if (!cache.has(id)) {
```

Comment:
This looks rather Pythonic to me :) Sadly, it won't work in JS. `id` is a new object literal every time this code runs. `cache.has(id)` will always be `false`.